explainer.md

```diff
@@ -532,6 +532,23 @@ function drawScene() {
 }
 ```
 
+### Changing the Field of View for inline sessions
+
+Whenever possible projection matrices delivered by the API should make use of physical properties such as the headset optics or camera lens to determine the field of view to use. However, for most inline content there aren't any physically based values to draw from. In order to provide a unified render pipeline for inline content an arbitrary field of view must be selected.
+
+By default a vertical field of view of 0.5 radians (90 degrees) is used for inline sessions. The horizontal field of view can be computed from the vertical field of view based on the width/height ratio of the `outputContext`'s canvas.
+
+If a different default field of view is desired, it can be specified by passing a new `verticalFieldOfView` value, in radians, to the `updateRenderState` method:
+
+```js
+// This changes the default vertical field of view to 0.4 radians (72 degrees).
+xrSession.updateRenderState({
+  verticalFieldOfView: 0.4 * Math.PI,
+});
+```
+
+The UA is allowed to clamp the value, and it is expected that if any physically-based values are available they should always be used in favor of the default. Attempting to set a `verticalFieldOfView` value on an immersive session will cause `updateRenderState()` to throw an `InvalidStateError`.
```

Comment:
> The UA is allowed to clamp the value, and it is expected that if any physically-based values are available they should always be used in favor of the default.

Two things here:
1. I'm inclined to have this be a stronger statement; if physically-based values are available, they MUST always be used in favor of the default.  Though perhaps I'm not recalling a reason for allowing this?
2. That word "default" again ;)